### PR TITLE
Fix compilation of ppx_test

### PR DESCRIPTION
**Pull request by samoht**
/cc @avsm @djs55

### CODE REVIEW

**Comment by djs55** [comment]
I'm a bit confused -- where does `cstruct-ppx` come from?

**Comment by samoht** [comment]
Fixed to match https://github.com/mirage/ocaml-cstruct/blob/master/ppx/jbuild#L5

**Comment by avsm** [comment]
that should be ppx_cstruct

File: ppx_test/with-sexp/jbuild [hunk 1]
```diff
@@ -2,11 +2,10 @@
 
 (executables
  ((names (ppx_cstruct_and_sexp))
-  (preprocess (pps (cstruct.ppx ppx_sexp_conv -no-check)))
+  (preprocess (pps (cstruct_ppx ppx_sexp_conv -no-check)))
```
Comment (avsm) [inline ppx_test/with-sexp/jbuild]: dont release with this -- it should be ppx_cstruct

Comment (samoht) [inline ppx_test/with-sexp/jbuild]: woops. Why does this pass on Travis?

Comment (samoht) [inline ppx_test/with-sexp/jbuild]: See https://github.com/mirage/ocaml-cstruct/pull/156

**Pull request merged**